src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Entity/EntityChangeRequestCommandHandler.php

```diff
@@ -28,30 +30,24 @@
 use Surfnet\ServiceProviderDashboard\Application\Service\TicketService;
 use Surfnet\ServiceProviderDashboard\Domain\Repository\EntityChangeRequestRepository;
 use Surfnet\ServiceProviderDashboard\Infrastructure\HttpClient\Exceptions\RuntimeException\PublishMetadataException;
-use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
 
 class EntityChangeRequestCommandHandler implements CommandHandler
 {
-    /**
-     * @var string
-     */
-    private $summaryTranslationKey;
+    private readonly string $summaryTranslationKey;
 
-    /**
-     * @var string
-     */
-    private $descriptionTranslationKey;
+    private readonly string $descriptionTranslationKey;
 
     public function __construct(
         private readonly EntityChangeRequestRepository $repository,
         private readonly EntityServiceInterface $entityService,
         private readonly TicketService $ticketService,
-        private readonly FlashBagInterface $flashBag,
+        private readonly RequestStack $requestStack,
         private readonly MailService $mailService,
         private readonly LoggerInterface $logger,
-        private readonly string $issueType
+        private readonly string $issueType,
     ) {
-        if (empty($issueType)) {
+        if ($issueType === '' || $issueType === '0') {
```

Comment:
This looks like a change in behavior; empty($issueType) is not the same as $issuType === '0' right?

Reply:
reverted this one, it was a rector change